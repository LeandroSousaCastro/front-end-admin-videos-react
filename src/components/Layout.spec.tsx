import { render } from "@testing-library/react";
import { Layout } from "./Layout";

describe("Header test", () => {
  it("should render correctly", () => {
    const { asFragment } = render(
      <Layout>
        <div>test</div>
      </Layout>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
