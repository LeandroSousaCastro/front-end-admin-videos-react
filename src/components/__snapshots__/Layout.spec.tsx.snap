// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test should render correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-0"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1f2kw6w-MuiContainer-root"
    >
      <div>
        test
      </div>
    </div>
  </div>
</DocumentFragment>
`;
